src/main/java/github/grace000/erversay/Constants/Headers.java

```diff
@@ -5,4 +5,5 @@
     public final static String OPTIONS_TWO_HEADER = "Allow: OPTIONS, GET, HEAD, PUT, POST";
     public final static String CONTENT_LENGTH = "content-length";
     public final static String NOT_ALLOWED_HEADER = "Allow: HEAD, OPTIONS";
+    public final static String REDIRECT_HEADER = "Location: http://127.0.0.1:5000/simple_get";
```

Comment:
Any reason this is hardcoded? `http://127.0.0.1:5000` And is there a plan _not_ to hard code it here?